### DISCUSSION TITLE
Uploading images with Twitter API v2?

### DISCUSSION THREAD

**Discussion by gordonbrillon** [tweepy/tweepy#1729] [Questions] [answered]
Hi there! I'm trying to use Tweepy to post tweets that will have new images attached. I want to specify a local file to upload, rather than pointing to a Twitter media id. Is there a way to do this using the Twitter API v2?

I see that you can use api.update_with_media to accomplish this with v1, is there something similar for v2?

Thank you, and sorry if this is obvious or I got my terminology wrong. I'm very new to this!

**Comment by Harmon758** [comment, marked as answer]
This isn't possible with Twitter API v2 yet.
It's [on the roadmap](https://trello.com/c/Zr9zDrJx/109-replacement-of-media-uploads-functionality).

**Reply by gordonbrillon** [reply]
Thanks for the response! I found the roadmap myself a little while after asking. Will press on with v1.
